js/windowEventRecorder.js

```diff
@@ -39,12 +39,10 @@ const recordEvent = (e) => {
 
         if (window.parent !== window) {
             let frame = window
-            let frameElement = window.frameElement
             let frameTag = window.frameElement ? window.frameElement.tagName.toLowerCase() : 'iframe';
 
             if (frame.name) {
                 attrs['frameName'] = frame.name
-                attrs['frameElement'] = frameElement
 
                 selector = `${frameTag}[name="${frame.name}"] ${selector}`
```

Comment:
Wow, cool! This works for me too. Not very clear **why** though

Reply:
I think Firefox just silently refused to send anything in the message (`sendMessage`) that couldn't be JSON-ified. So it was dropping them when an `Element` was included. Took me a while to track it down to that.